<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoopW</class>
 <widget class="QWidget" name="LoopW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>830</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="hl1" stretch="1,10,1,1">
     <item>
      <widget class="QLabel" name="laNameT">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="laName">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbSetName">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Set</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hl2">
     <item>
      <widget class="QLabel" name="laLoopStyle">
       <property name="text">
        <string>Loop style</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbLoopStyle">
       <item>
        <property name="text">
         <string>Fixed</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Random [every instance]</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Shuffle [every instance]</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fixed [random start]</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fixed [shift start at each instance]</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Random [one for all instances]</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Shuffle [one for all instances]</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Random [every N instances]</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Shuffle [every N instances]</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>File</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="cbNoFollowingValues">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>No following values (random and shuffle)</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>132</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hl3" stretch="1,1,0,0,0">
     <item>
      <widget class="QLabel" name="laNbReps">
       <property name="text">
        <string>Number of repetitions</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbNbReps">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>N</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbN">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wSetList" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>300</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1,0,10">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="wAddSet" native="true">
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="laNewSet">
           <property name="text">
            <string>Sets to add (one per line)</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="pbAdd">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="4">
          <widget class="QTextEdit" name="teAdd"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="wChooseFile" native="true">
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="1">
          <widget class="QPushButton" name="pbReload">
           <property name="text">
            <string>Reload</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4" rowspan="4">
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>366</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="pbChooseFile">
           <property name="text">
            <string>Choose file</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="4">
          <widget class="QLabel" name="laCurrentFile">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="laSetList">
        <property name="text">
         <string>Set list</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="twSets">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="showGrid">
         <bool>true</bool>
        </property>
        <property name="rowCount">
         <number>1</number>
        </property>
        <property name="columnCount">
         <number>2</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <row/>
        <column/>
        <column/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="fMove">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbUp">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Up</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbRemove">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Del</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbDown">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Down</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Informations:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="teInformations"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
